pkg/config/cloud/powervs/powervs_regions.go

```diff
@@ -0,0 +1,102 @@
+package powervs
+
+import (
+	"fmt"
+)
+
+// Since there is no API to query these, we have to hard-code them here.
+
+// Region describes resources associated with a region in Power VS.
+// We're using a few items from the IBM Cloud VPC offering. The region names
+// for VPC are different so another function of this is to correlate those.
+type Region struct {
+	Description string
+	VPCRegion   string
+	Zones       []string
+}
+
+// Regions provides a mapping between Power VS and IBM Cloud VPC regions.
+var Regions = map[string]Region{
```

Comment:
This looks like configuration data, not something we'd want to keep in a repository. 
/hold

Reply:
@soltysh For some context this is a mapping between two different cloud services in IBM Cloud. Currently this is in the openshift-installer, but we found that it would need to be referenced by other projects as well. It was then suggested to put it here in a comment in this PR https://github.com/openshift/api/pull/1383

If something like this doesn't belong here, could you suggest where it should go?